src/software/world/robot.h

```diff
@@ -248,12 +248,20 @@ class Robot
     Timestamp getMostRecentTimestamp() const;
 
     /**
-     * Returns the hardware capabilities of the robot
+     * Returns the missing capabilities of the robot
      *
-     * @return the hardware capabilities of the robot
+     * @return the missing capabilities of the robot
      */
     const std::set<RobotCapabilities::Capability> &getCapabilitiesBlacklist() const;
 
+    // TODO: test me!
```

Comment:
Todo?

Reply:
Whoops, that was sloppy. Added tests for getting robotcapabilities whitelist and blacklist.